<launch>
  <remap from="/calibration_publisher/image_raw" to="/quori/face/image" />

  <node name="face" pkg="quori_face" type="quori_face_node" ns="/quori" output="screen" respawn="false" required="true">
    <env name="DISPLAY" value=":0" />
    <rosparam file="$(find quori_face)/config/params.yaml" command="load"/>
    <param name="update_params" type="bool" value="true" />
  </node>
  <node name="dynamic_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" ns="/quori/face" output="screen" respawn="false" required="true">
  </node>
  <node name="calibration_publisher" pkg="image_publisher" type="image_publisher" output="screen" respawn="false" required="true" args="$(find quori_face)/assets/calibration_helper.png">
  </node>
</launch>